<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
                background: qlineargradient(
                    x1:0, y1:0, x2:0, y2:1,
                    stop:0 #e0f7fa,       /* top: soft pastel cyan */
                    stop:0.5 #f9f9f9,     /* middle: very light gray */
                    stop:1 #ffffff        /* bottom: pure white */
                );
            }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
        background: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #ffffff,
            stop:1 #e6e6e6
        );
        border-radius: 15px;
        border: 1px solid #d9d9d9;
    }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="statButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
        background-color: #3498db;  /* Button color */
        color: white;               /* Text color */
        border-radius: 15px;        /* Rounded corners */
        padding: 10px 20px;         /* Padding */
    }
    QPushButton:hover {
        background-color: #2980b9;  /* Hover color */
    }</string>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cashierButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
        background-color: #3498db;  /* Button color */
        color: white;               /* Text color */
        border-radius: 15px;        /* Rounded corners */
        padding: 10px 20px;         /* Padding */
    }
    QPushButton:hover {
        background-color: #2980b9;  /* Hover color */
    }</string>
     </property>
     <property name="text">
      <string>Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
        background-color: #3498db;  /* Button color */
        color: white;               /* Text color */
        border-radius: 15px;        /* Rounded corners */
        padding: 10px 20px;         /* Padding */
    }
    QPushButton:hover {
        background-color: #2980b9;  /* Hover color */
    }</string>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="promotionButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
        background-color: #3498db;  /* Button color */
        color: white;               /* Text color */
        border-radius: 15px;        /* Rounded corners */
        padding: 10px 20px;         /* Padding */
    }
    QPushButton:hover {
        background-color: #2980b9;  /* Hover color */
    }</string>
     </property>
     <property name="text">
      <string>Promotions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
        background-color: #3498db;  /* Button color */
        color: white;               /* Text color */
        border-radius: 15px;        /* Rounded corners */
        padding: 10px 20px;         /* Padding */
    }
    QPushButton:hover {
        background-color: #2980b9;  /* Hover color */
    }</string>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
    <widget class="QLabel" name="greeting_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
        background: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #ffffff,
            stop:1 #e6e6e6
        );
        border-radius: 15px;
        border: 1px solid #d9d9d9;
    }</string>
     </property>
     <property name="text">
      <string>Promotions</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #ffffff, stop:1 #e0e0e0
        );
        border: 1px solid #b3b3b3;
        border-radius: 14px; /* half of height for fully rounded */
        color: #333333;
        font-weight: bold;
        font-size: 12px;
    }
    QPushButton:hover {
        background: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #f2f2f2, stop:1 #d9d9d9
        );
    }
    QPushButton:pressed {
        background: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #d9d9d9, stop:1 #bfbfbf
        );
    }</string>
    </property>
    <property name="text">
     <string>logout</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="promotionTable">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
        background-color: #ffffff;
        border: 2px solid #b0b0b0;  /* soft gray border */
        border-radius: 12px;         /* rounded corners */
        gridline-color: #e0e0e0;     /* subtle grid lines */
        padding: 5px;
    }
    QHeaderView::section {
        background-color: #f5f5f5;   /* light header */
        border: 1px solid #d0d0d0;
        border-radius: 6px;          /* rounded header corners */
        padding: 4px;
        font-weight: bold;
    }
    QTableWidget::item {
        padding: 6px;
    }
    QTableWidget::item:selected {
        background-color: #cce7ff;  /* gentle selection color */
        color: #000000;
    }</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="promotionNameForm">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
        border: 2px solid #cccccc;           /* soft gray border */
        border-radius: 15px;                 /* very round corners */
        padding: 6px 12px;                   /* comfortable spacing inside */
        background-color: #fdfdfd;           /* very light background */
        selection-background-color: #cce7ff; /* gentle blue selection */
        font-size: 14px;                      /* slightly bigger text */
    }
    QLineEdit:focus {
        border: 2px solid #7ab8f5;           /* highlight when focused */
        background-color: #ffffff;
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>469</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Promotion Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="promotionCodeForm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
        border: 2px solid #cccccc;           /* soft gray border */
        border-radius: 15px;                 /* very round corners */
        padding: 6px 12px;                   /* comfortable spacing inside */
        background-color: #fdfdfd;           /* very light background */
        selection-background-color: #cce7ff; /* gentle blue selection */
        font-size: 14px;                      /* slightly bigger text */
    }
    QLineEdit:focus {
        border: 2px solid #7ab8f5;           /* highlight when focused */
        background-color: #ffffff;
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>468</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Promotion Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>469</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Discount %</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPromo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #7e57c2;       /* friendly purple */
        color: white;                     /* text color */
        border: none;
        border-radius: 8px;               /* moderately round */
        padding: 8px 20px;                /* comfortable size */
        font-size: 14px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #9575cd;       /* slightly lighter purple on hover */
    }
    QPushButton:pressed {
        background-color: #5e35b1;       /* darker purple when pressed */
    }</string>
    </property>
    <property name="text">
     <string>New Promo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="discountForm">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
        border: 2px solid #cccccc;           /* soft gray border */
        border-radius: 15px;                 /* very round corners */
        padding: 6px 12px;                   /* comfortable spacing inside */
        background-color: #fdfdfd;           /* very light background */
        selection-background-color: #cce7ff; /* gentle blue selection */
        font-size: 14px;                      /* slightly bigger text */
    }
    QLineEdit:focus {
        border: 2px solid #7ab8f5;           /* highlight when focused */
        background-color: #ffffff;
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Purchase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minimumPurchaseForm">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
        border: 2px solid #cccccc;           /* soft gray border */
        border-radius: 15px;                 /* very round corners */
        padding: 6px 12px;                   /* comfortable spacing inside */
        background-color: #fdfdfd;           /* very light background */
        selection-background-color: #cce7ff; /* gentle blue selection */
        font-size: 14px;                      /* slightly bigger text */
    }
    QLineEdit:focus {
        border: 2px solid #7ab8f5;           /* highlight when focused */
        background-color: #ffffff;
    }</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMsg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[ERROR MESSAGE]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPromo">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #7e57c2;       /* friendly purple */
        color: white;                     /* text color */
        border: none;
        border-radius: 8px;               /* moderately round */
        padding: 8px 20px;                /* comfortable size */
        font-size: 14px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #9575cd;       /* slightly lighter purple on hover */
    }
    QPushButton:pressed {
        background-color: #5e35b1;       /* darker purple when pressed */
    }</string>
    </property>
    <property name="text">
     <string>Edit Promo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePromo">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #7e57c2;       /* friendly purple */
        color: white;                     /* text color */
        border: none;
        border-radius: 8px;               /* moderately round */
        padding: 8px 20px;                /* comfortable size */
        font-size: 14px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #9575cd;       /* slightly lighter purple on hover */
    }
    QPushButton:pressed {
        background-color: #5e35b1;       /* darker purple when pressed */
    }</string>
    </property>
    <property name="text">
     <string>Delete Promo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
